reimbursable_with_tax/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,13 @@
+This module adds the option to add tax on tab reimbursables.
+
+Reimbursables are payments for services that your supplier has made on behalf
```

Comment:
We don't need the original description here, just what is added is enough (i.e., about tax)